stags/src/test/scala/co/pjrt/stags/TagLineTest.scala

```diff
@@ -59,8 +59,9 @@ class TagLineTest extends FreeSpec with Matchers {
       TagLine(tag, filePath)
         .relativize(target)
         .filePath
-        .toString shouldBe expectedS
-
+        .toString
+        .replaceAllLiterally("\\","/") //windows path sep is backwards
```

Comment:
I don't understand this change

Reply:
Oddly, all the changes are not shown there.  The full diff can be seen in [line 64](https://github.com/drxcc/stags/blob/8fda376bca1dfc286afd847752202f9a4acbd01f/stags/src/test/scala/co/pjrt/stags/TagLineTest.scala#L64)

the `replaceAllLiterally` is just wedged before the `shouldBe`

```
.toString
.replaceAllLiterally("\\","/") //windows path sep is backwards
.shouldBe(expectedS)
```

Reply:
I get that, I don't get why this was needed. Are u running this test on windows and it is failing?

Reply:
I was testing this on a Windows machine.  In order to make the test work, I had to make the paths backward paths match the tests results.

the `replaceAllLiterally` is just wedged in before the `shouldBe` .